pipelines/common/src/main/java/org/openmrs/analytics/JdbcConnectionUtil.java

```diff
@@ -13,20 +13,21 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 package org.openmrs.analytics;
 
 import com.google.common.base.Preconditions;
 import com.mchange.v2.c3p0.ComboPooledDataSource;
 import java.beans.PropertyVetoException;
+import java.io.Serializable;
 import java.sql.Connection;
+import java.sql.PreparedStatement;
 import java.sql.SQLException;
 import java.sql.Statement;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class JdbcConnectionUtil {
-
+public class JdbcConnectionUtil implements Serializable {
```

Comment:
I am guessing that you needed this to be serializable because it is now being passed to DoFns, correct? If yes, then is it okay to serialize an instance of this that is created on a different machine and use it on another one (so not different threads or processes on the same machine but different ones)? Because that is needed when passing objects to DoFns (they might be on a different machine).

Reply:
Yes, that is the reason why I had to make it serializable.

Reply:
By default this instance variable should be serialised, So this comment can be ignored. I am resolving this.